### PR TITLE
context withoutCancel

### PR DESCRIPTION
**Pull request by c0dehu1k**
Fixes issue-42

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1439,6 +1439,9 @@ When the response has been written to the client, the context associated with th
 
 In the latter case, calling publish will return an error because we returned the HTTP response quickly.
 
+Note: From Go 1.21, there is a way to create a new context without cancel. WithoutCancel returns a copy of parent that is not canceled when parent is canceled.
```
Comment (teivah) [inline docs/index.md:1442]: Can you please use this note syntax? https://github.com/teivah/100-go-mistakes/blob/master/docs/index.md?plain=1#L282-L284

Comment (teivah) [inline docs/index.md:1442]: WithoutCancel => [`context.WithoutCancel`](https://pkg.go.dev/context#WithoutCancel), please

File: docs/index.md [hunk 2]
```diff
@@ -1439,6 +1439,9 @@ When the response has been written to the client, the context associated with th
...
+Read more [here](https://pkg.go.dev/context#WithoutCancel)
```
Comment (teivah) [inline docs/index.md:1443]: Let's remove this line, please (the link will be attached to context.WithoutCancel)

**Review by teivah — CHANGES_REQUESTED** [review]

**Comment by c0dehu1k** [comment]
@teivah Thanks for the comments, I have fixed them all. Please have a look.

**Review by teivah — APPROVED** [review]
Thank you

**Pull request merged**